### DISCUSSION TITLE
Loading a dynamic javascript file

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#13237] [Help]
### Package

Panel builder

### Package Version

3.2.90 

### How can we help you?

https://filamentphp.com/docs/3.x/support/assets#registering-javascript-files

I would like to register a user's individual javascript script on a filament page, and the documentation says that we have to call `FilamentAsset::register(...)` in `boot()`.

However in `boot()`, `auth()->user()` is null. and the user's individual script is named something like `/js/script-{$user->id}.js`.

Any ideas how to go about it?

**Comment by bilogic** [comment]
There seems to be a problem everytime I try to post a discussion, my OP goes missing...

**Reply by leandrocfe** [reply]
Would you like to use a dynamic js based on users?
Could you provide more info?

**Reply by bilogic** [reply]
are you able to see my OP? what kind of additional info do you need?
